src/Apps/Artwork/Components/Seo/__tests__/SeoDataForArtwork.jest.tsx

```diff
@@ -0,0 +1,388 @@
+      })
+
+      it("Does not render seller within offer when profile image (required) is not present", () => {
+        renderWithRelay({
+          Artwork: () => ({
+            ...defaultArtwork,
+            partner: {
+              id: "partner-id",
+              name: "Wright",
+              type: "Gallery",
+              profile: {
+                id: "profile-id",
+                image: null,
+              },
+            },
+            listPrice: {
+              __typename: "Money",
+              major: 1000,
+              currencyCode: "USD",
+            },
+          }),
+        })
+
+        const data = getSchemaData()
+        expect(data.offers.seller).toBeUndefined()
+      })
+    })
+
+    describe("Artwork dimensions", () => {
+      it("renders no dimensions when dimensions aren't parseable", () => {
+        renderWithRelay({
+          Artwork: () => ({
+            ...defaultArtwork,
+            dimensions: {
+              in: "one point twenty one gigawatts",
+              cm: null,
+            },
+          }),
+        })
+
+        const data = getSchemaData()
+        expect(data.width).toBeUndefined()
+        expect(data.height).toBeUndefined()
+        expect(data.depth).toBeUndefined()
+      })
+
+      it("renders width and height when given two dimensions", () => {
+        renderWithRelay({
+          Artwork: () => ({
+            ...defaultArtwork,
+            dimensions: {
+              in: "2 × 4 in",
+              cm: null,
+            },
+          }),
+        })
+
+        const data = getSchemaData()
+        expect(data.width).toEqual({
+          "@type": "Distance",
+          name: "2 in",
+        })
+        expect(data.height).toEqual({
+          "@type": "Distance",
+          name: "4 in",
+        })
+        expect(data.depth).toBeUndefined()
+      })
+
+      it("renders width, height, and depth when given three dimensions", () => {
+        renderWithRelay({
+          Artwork: () => ({
+            ...defaultArtwork,
+            dimensions: {
+              in: "2 × 4 × 6 in",
+              cm: null,
+            },
+          }),
+        })
+
+        const data = getSchemaData()
+        expect(data.width).toEqual({
+          "@type": "Distance",
+          name: "2 in",
+        })
+        expect(data.height).toEqual({
+          "@type": "Distance",
+          name: "4 in",
+        })
+        expect(data.depth).toEqual({
+          "@type": "Distance",
+          name: "6 in",
+        })
+      })
+
+      it("successfully handles case when no dimensions are present", () => {
+        renderWithRelay({
+          Artwork: () => ({
+            ...defaultArtwork,
+            dimensions: null,
+          }),
+        })
+
+        const data = getSchemaData()
+        expect(data.width).toBeUndefined()
+        expect(data.height).toBeUndefined()
+        expect(data.depth).toBeUndefined()
+      })
+    })
+  })
+})
```

Comment:
Seems that enzyme specs that use react-head need to be modernized due to react 18